conf/messages.en

```diff
@@ -429,20 +429,35 @@ ukDividendsSharesLoans.error.required = Select ukDividendsSharesLoans
 ukDividendsSharesLoans.agent.error.required = Select ukDividendsSharesLoans
 ukDividendsSharesLoans.change.hidden = UkDividendsSharesLoans
 
-ukInsuranceGains.title = ukInsuranceGains
-ukInsuranceGains.agent.title = ukInsuranceGains
-ukInsuranceGains.heading = ukInsuranceGains
-ukInsuranceGains.agent.heading = ukInsuranceGains
-ukInsuranceGains.option1 = Option 1
-ukInsuranceGains.option2 = Option 2
-ukInsuranceGains.agent.option1 = Option 1
-ukInsuranceGains.agent.option2 = Option 2
-ukInsuranceGains.exclusive = Exclusive
-ukInsuranceGains.agent.exclusive = Exclusive
+ukInsuranceGains.title = UK insurance gains
```

Comment:
It may be worth adding these keys with the English values to the Welsh messages file as placeholders until we get the translations.